Source/swiftlint/Commands/ShowDocsCommand.swift

```diff
@@ -0,0 +1,29 @@
+import Commandant
+import Foundation
+
+struct ShowDocsCommand: CommandProtocol {
+    let verb = "show-docs"
+    let function = "Open `SwiftLint Framework Docs` on web browser"
```

Comment:
no need for the backquotes (and I would remove the "Framework" from the sentence)

Reply:
Fixed it!